src/foam/nanos/auth/UserQueryService.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.auth',
+  name: 'UserQueryService',
+
```

Comment:
documentation, not just this class, but how the query service is intended to be used.

Reply:
Done.